SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -113,9 +114,16 @@ void USpatialWorkerConnection::SendComponentUpdate(Worker_EntityId EntityId, FWo
 }
 
 Worker_RequestId USpatialWorkerConnection::SendCommandRequest(Worker_EntityId EntityId, Worker_CommandRequest* Request, uint32_t CommandId,
-															  const TOptional<Trace_SpanId>& SpanId)
+															  bool EnableRetry = false, const TOptional<Trace_SpanId>& SpanId)
```

Comment:
could this argument instead be the retry policy directly? Right now it is unclear without a little dig what the boolean at the callsite is, and given we want to retry differently for the different commands in this PR (i.e., some false some true) it may make sense to make this flexible now

Reply:
+1 I think for regular commands especially you might want to adjust things.

Reply:
I think this goes for the other ones too. It's probably clearer from the call site to see it use the RETRY_UNTIL_COMPLETE that it is to see `true` even if they never actually use a different retry handler for things like reserving entity IDs.